src/keylessBackup/slice.ts

```diff
@@ -0,0 +1,41 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit'
+
+export interface State {
+  google: {
+    loading: boolean
+    idToken: string | null
```

Comment:
Is it safe to persist `idToken`? The persisted state has no protection and is readable in clear text.

Reply:
The entire keylessBackup state is blacklisted here: https://github.com/valora-inc/wallet/pull/3741/files#diff-b3695b5171edbee1bb22dc8893e3517773bacdd8a51ba2a568e077f6e1e07940R29, so this wouldn't be persisted right?